metaflow/plugins/aip/aip.py

```diff
@@ -889,7 +888,42 @@ def _set_container_labels(self, container_op: ContainerOp):
                 raise ValueError(
                     f"Tag name {annotation_name} must be no more than 63 characters"
                 )
-            container_op.add_pod_annotation(annotation_name, annotation_value)
+            ret[annotation_name] = annotation_value
+
+        # - In context of Zillow CICD self.username == "cicd_compile"
+        # - In the context of a Zillow NB self.username == METAFLOW_USER (user_alias)
+        # - In the context of Metaflow integration tests self.username == USER=$GITLAB_USER_EMAIL
+        owner = self.username
+        if "@" in owner:
+            owner = owner.split("@")[0]
+        ret["zodiac.zillowgroup.net/owner"] = owner
+
+        # Add in Zodiac service and team labels to the aip pods if the environment variable is
+        # present in the notebook (individual profile notebooks only) and set them. These labels
+        # are not being added by poddefaults as they were removed. Workflows launched in project
+        # profiles still get these labels added via poddefaults. Also adds in logging topic
```

Comment:
Sorry maybe I'm not following here, are we still relying on PodDefaults or is this done via the Webhook now?

Reply:
this is needed to support zodiac service per notebook, that is why it is ignored in the webhook

Reply:
Thanks @aaron-arellano , I'll add that to the code comments, because we forget this context!

Reply:
done, with Aaron's comment and ChatGPT help!